src/game_theory/sprague-grundy-nim.md

```diff
@@ -114,13 +114,13 @@ Since the game is acyclic, sooner or later the current player won't be able to u
 
 ### Sprague-Grundy theorem
 
-Let's consider a state $v$ of a two-player impartial game and let $v_i$ be the states reachable from it (where $i \in \{ 1, 2, \dots, k \} , k \ge 0$).
+Let's consider a state $v$ of a two-player impartial game and let $v_i$ be the states reachable from it (where $i \in \\{ 1, 2, \dots, k \\} , k \ge 0$).
```

Comment:
I'm sorry, but this change is not good.
The markdown-to-html converter that we use has a but, that removes a backslash whenever a backslash appears directly in front of a special character. So we do need two backslashes here. For more infos: https://github.com/e-maxx-eng/e-maxx-eng/issues/236

Reply:
Btw, it's possible to test the output of our converter at: https://cp-algorithms.com/test.php

Reply:
I'm not sure what you mean... I changed one backslash to two backslashes, precisely so that the brackets would appear.

Reply:
Oh, sorry. I completely read it the wrong way.